doc/dev/index.rst

```diff
@@ -1337,21 +1337,21 @@ manually.
 However, there are times when the automated tests do not cover a particular
 scenario and manual testing is desired. It turns out that it is simple to
 adapt a test to stop and wait after the Ceph installation phase, and the
-engineer can then ssh into the running cluster.
+engineer can then ssh into the running cluster. Simply add the following
+snippet in the desired place within the test YAML and schedule a run with the
+test::
 
-This common use case is currently provided for by the following command::
+    - exec:
+        client.0:
+          - sleep 1000000000 # forever
```

Comment:
i am able to have the interactive python shell using following command in sepia.
```
teuthology ~/test.yaml --lock --machine-type mira
```

where `test.yaml` is

```yaml
roles:
    - [client.0]
repo: git://git.ceph.com/ceph.git
tasks:
    - interactive:
```